core/src/avm2/globals/flash/system/MessageChannel.as

```diff
@@ -1,11 +1,11 @@
 package flash.system {
     import flash.events.EventDispatcher;
+    import __ruffle__.stub_method;
 
     [API("682")]
-    [Ruffle(Abstract)]
     public final class MessageChannel extends EventDispatcher {
```

Comment:
A stubbed constructor should be added with a comment that it should throw

`ArgumentError: Error #2012: MessageChannel$ class cannot be instantiated.`